README.rst

```diff
@@ -1,4 +1,4 @@
-Girder Worker |build-status| |docs-status| |license-badge| |gitter-badge|
+|logo| Girder Worker |build-status| |docs-status| |license-badge| |gitter-badge|
```

Comment:
Likely the line below (`=====`) needs to be updated to match the new length of this line. Otherwise this line isn't treated as a section header.